src/pipeline/postprocess/buildBQLTree.ts

```diff
@@ -10,6 +10,13 @@ import type { Entity, PipelineOperation } from '../pipeline';
 function isStringOrHasShow<TValue extends { $show?: boolean }>(value: TValue | string): value is TValue {
   return typeof value === 'string' || !!value.$show;
 }
+// TODO: temp solution
+const isOne = (children: any[], $id: string) => {
```

Comment:
This should happen obly when any of these:
- The field is cardianity ONE (So we now the received data as well)
- Anything in the query specified for this specific level a single value, in general because the idField has been defined, and its a single value ( so the output is also a single value, we don't expect an array)